### ISSUE TITLE
Conflicts are handled badly

### ISSUE THREAD

**Issue by luk3yx** [liberodark/ODrive#83]
**Issue:**

ODrive appears to prefer remote (Google Drive) changes over local changes.

**Steps to reproduce:**

1. Start ODrive.
1. Create a local folder with a text file inside it.
1. Stop ODrive.
1. Delete the folder from Google Drive.
1. Modify the text file locally.
1. Start ODrive again.

*Note that I cannot test it any further easily as ODrive will delete local data when started.*

**Comment by liberodark** [comment]
Yes is not a bug is only solution is add a ask for keep file or not !
